### DISCUSSION TITLE
How do I open terminals using Cmd + P `term 1` `term 2` `term 3`?

### DISCUSSION THREAD

**Discussion by Blankeos** [zed-industries/zed#19487] [Feature Requests]
In VSCode, it's very easy for me to open terminals, new terminals, and switch between existing terminals using cmd + p then writing `term 1`.

I know this doesn't exist in Zed atm, but is it possible to add it in my configs? Preferably maybe a custom `Cmd + Shift P` command?

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support

**Comment by Blankeos** [comment]
Is it possible to reopen this as a Feature Request? I noticed since it isn't possible yet... Just came back to this because recently coding w/ a bunch of terminals open has been a bit more important.

It would be very beneficial to add since you can have a bunch of Claude Code / OpenCode sessions just running at the same time. It's very hard to switch between them via the standard navigation shortcuts: `cmd+option ⬅️➡️`, etc.
